lib/messagebird/verify.rb

```diff
@@ -1,20 +1,25 @@
 # frozen_string_literal: true
 
 require 'time'
-
 require 'messagebird/base'
 
 module MessageBird
   class Verify < MessageBird::Base
-    attr_accessor :id, :recipient, :reference, :status, :href
-    attr_reader :created_datetime, :valid_until_date_time
+    attr_accessor :id,
+                  :href,
+                  :recipient,
+                  :reference,
+                  :messages,
+                  :status
+    attr_reader   :created_datetime,
+                  :valid_until_datetime
 
     def created_datetime=(value)
       @created_datetime = value_to_time(value)
     end
 
-    def valid_until_date_time=(value)
```

Comment:
Isn't a breaking change? Why it's necessary?

Reply:
Hey @jalerson, the gem generates the attribute names based on the response from the API, and it converts `camelCase` to `snake_case`.

The response from the server contains the `validUntilDatetime` key, therefore the `valid_until_date_time` attribute breaks.

It is a breaking change that fixes a broken gem, and this is a major release as well, so I think it fits.